<launch>
    <group ns="robot_bringup">
	    <node pkg="reinovo_control" type="blank" name="blank" output="screen" required="true"/>
	</group>
	<!--以上勿动-->


	<include file="$(find oryxbot_base)/launch/oryxbot_base.launch" /> 
	<include file="$(find ydlidar_ros)/launch/X2L.launch" />

</launch>
